packages/dev/core/src/Misc/sceneSerializer.ts

```diff
@@ -149,7 +164,7 @@ export class SceneSerializer {
         }
 
         //Physics
-        if (scene.isPhysicsEnabled()) {
+        if (scene.isPhysicsEnabled && scene.isPhysicsEnabled()) {

```

Comment:
nit: `if (scene.isPhysicsEnabled?.()) {` (it saves some characters for the planet!)